keel-artifact/src/test/kotlin/com/netflix/spinnaker/keel/artifacts/DebianArtifactSupplierTests.kt

```diff
@@ -174,6 +182,11 @@ internal class DebianArtifactSupplierTests : JUnit5Minutests {
           .isTrue()
       }
 
+      test ("should process artifact successfully2") {
```

Comment:
```suggestion
      test ("should process artifact successfully") {
```

Reply:
I have 2 tests with the same name, so this is 2 😄  will fix later on